src/main/kotlin/org/kamiblue/client/module/modules/movement/Step.kt

```diff
@@ -30,7 +30,7 @@ internal object Step : Module(
     modulePriority = 200
 ) {
     private val mode by setting("Mode", Mode.PACKET)
-    private val upStep = setting("Up Step", true)
+    private val upStep = setting("Up Step", true, consumer = { _, value -> setBaritoneAssumeStep(); value })
```

Comment:
You should be using a valueListener for this instead of consumer

Reply:
That name was kind of misleading and I should probably change it to something else. Do you have any idea?

Reply:
Not sure about a better name, I think "Up Step" sounds alright because there is also  "Down Step", and changing just one of them might make it more confusing.

I will update the code soon.

Reply:
> Not sure about a better name, I think "Up Step" sounds alright because there is also "Down Step", and changing just one of them might make it more confusing.
> 
> I will update the code soon.

No I mean the name of consumer was misleading